src/basic/path-util.c

```diff
@@ -525,6 +525,18 @@ int path_compare_filename(const char *a, const char *b) {
         return strcmp(fa, fb);
 }
 
+int path_equal_or_inode_same_checked(const char *a, const char *b, int flags) {
```

Comment:
Please call this `path_equal_or_inode_same_full` or so. "checked" is harder to understand.

Reply:
Done.